containers/verifier/build-dependencies.txt

```diff
@@ -0,0 +1,16 @@
+libssl-dev
+git
+pkg-config
+libffi-dev
+software-properties-common
```

Comment:
Could we have a separate file just for pyenv/python dependencies that's the same in every container? It would make managing these dependencies easier.